examples/Geoportal.ipynb

```diff
@@ -18,45 +18,57 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "119aacd7-fb6b-4bf5-9a97-ac8301eabfed",
+   "id": "68a99b3b-58f1-4251-a724-f5135ac72be1",
    "metadata": {},
    "outputs": [],
    "source": [
     "from ipyleaflet import *\n",
     "from ipyleaflet.geoportal import GeoportalWMTS\n",
-    "m1 = Map(center=(48.799999, 2.26667), zoom=10, zoom_control=True)\n",
+    "m1 = Map(center=(44.799999, 2.26667), zoom=10, zoom_control=True)\n",
     "layer1 = GeoportalWMTS() #default\n",
     "m1.add(layer1)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "2936c8dc-1546-4f13-8c18-6984aaf1c0e3",
+   "id": "9452eaf1-38ff-463f-b272-95baa8f840a5",
    "metadata": {},
    "outputs": [],
    "source": [
-    "m2 = Map(center=(44.400002, 3.36667), zoom=8, zoom_control=True)\n",
+    "m2 = Map(center=(44.400002, 3.36667), zoom=10, zoom_control=True)\n",
     "layer2 = GeoportalWMTS(layer = 'ELEVATION.SLOPES', api_key = 'altimetrie')\n",
     "m2.add(layer2)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "cfeff952-288f-4e56-9dd7-4b7e160e6305",
+   "id": "69d5a707-f501-4d5c-a22e-3e59e2656aaf",
    "metadata": {},
    "outputs": [],
    "source": [
-    "m3 = Map(center=(48.799999, 2.26667), zoom=10, zoom_control=True)\n",
-    "layer3 = GeoportalWMTS(layer = 'GEOGRAPHICALGRIDSYSTEMS.PLANIGNV2', api_key = 'cartes')\n",
+    "m3 = Map(center=(44.799999, 2.26667), zoom=10, zoom_control=True)\n",
+    "layer3 = GeoportalWMTS(layer = 'GEOGRAPHICALGRIDSYSTEMS.PLANIGNV2', api_key = 'cartes', format = 'image/png')\n",
```

Comment:
Should the `format` and `api_key` be set automatically, depending on the `layer`?